src/RobotsHeaders.php

```diff
@@ -41,12 +41,50 @@ public function mayFollow(string $userAgent = '*'): bool
 
     public function noindex(string $userAgent = '*'): bool
     {
-        return $this->robotHeadersProperties[$userAgent]['noindex'] ?? false;
+        $userAgent = strtolower($userAgent);
+
+        if (isset($this->robotHeadersProperties[$userAgent])
+            && isset($this->robotHeadersProperties[$userAgent]['noindex'])) {
+            return $this->robotHeadersProperties[$userAgent]['noindex'];
+        }
+
+        if ($wildCard = $this->checkForWildcard($userAgent, 'noindex') !== null) {
```

Comment:
`$wildCard` does not contain a `wildcard`. `$wildcardFound` is a better name.

Reply:
Code removed.